composer.json

```diff
@@ -13,9 +13,10 @@
         "vimeo/psalm": "dev-master"
     },
     "require-dev": {
-        "phpunit/phpunit": "^7.2",
+        "phpunit/phpunit": "^9.5.5",
```

Comment:
@weirdan
  Problem 1
    - Root composer.json requires phpunit/phpunit ^9.5.5 -> satisfiable by phpunit/phpunit[9.5.5].
    - phpunit/phpunit 9.5.5 requires php >=7.3 -> your php version (7.2.24) does not satisfy that requirement.

Composer stated PHP >=7.1 is required. I have changed this back to 7.2 for simplicity in my environment as it doesn't impact my tests.